src/views/Dashboard.vue

```diff
@@ -0,0 +1,112 @@
+<!--
+  - @copyright Copyright (c) 2020 Georg Ehrke <oc.list@georgehrke.com>
+  - @author Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<DashboardWidget
+		id="announcementcenter_panel"
+		:items="items"
+		:loading="loading">
+		<template v-slot:default="{ item }">
+			<DashboardWidgetItem :item="item">
+				<template v-slot:avatar>
+					<Avatar
+						:user="item.authorUserId"
+						:display-name="item.authorUsername"
+						:size="44"
+						:show-user-status="false" />
```

Comment:
This is also a bit weird. I had to do this because otherwise the user status would show for the announcement, even when overlayIconUrl was used. Maybe in that case we can forward to the Avatar component to not show the status, @juliushaertl

Reply:
Fixed with https://github.com/nextcloud/nextcloud-vue-dashboard/pull/40 so will come with `@nextcloud/vue-dashboard` 0.1.9